recipe/bld.bat

```diff
@@ -0,0 +1,5 @@
+mkdir build
+cd build
+
+cmake -G "NMake Makefiles" -D CMAKE_INSTALL_PREFIX=%LIBRARY_PREFIX% ..
```

Comment:
Probably needs this:
```suggestion
cmake -G "NMake Makefiles" -D CMAKE_INSTALL_PREFIX=%LIBRARY_PREFIX% -DCMAKE_INSTALL_LIBDIR=%LIBRARY_LIB% ..
```

Reply:
Also likely want `-D CMAKE_BUILD_TYPE=Release` too:
```suggestion
cmake -G "NMake Makefiles" -D CMAKE_INSTALL_PREFIX=%LIBRARY_PREFIX% -D CMAKE_INSTALL_LIBDIR=%LIBRARY_LIB% -D CMAKE_BUILD_TYPE=Release ..
```

Reply:
Did you mean to mark this as resolved or happened it by accident?

Reply:
Maybe we could also use `CMAKE_PREFIX_PATH`? If `include`/`lib` are also the same defaults on Windows, this might work.
```suggestion
cmake ^
  -G "NMake Makefiles" ^
  -D CMAKE_BUILD_TYPE=Release ^
  -D CMAKE_INSTALL_PREFIX=%LIBRARY_PREFIX% ^
  -D CMAKE_PREFIX_PATH=%LIBRARY_PREFIX% ^
  ..
```

Reply:
(Sorry, for the mess of these multiple/single suggestions. Windows-specifics are buried deep in my head and I tend to forget them again and again.)

Reply:
I was doing the same thing as you with the release build type.